architecting-cloud-native-dotnet-apps-for-azure/cloud-native-devops.md

```diff
@@ -5,8 +5,246 @@ ms.date: 06/30/2019
+
+Through good DevOps practices it is possible to realize the advantages of Cloud Native Applications without suffocating under a mountain of work actually operating the applications. 
+
+There is no golden hammer when it comes to DevOps, nobody can sell a complete and all encompassing solution to releasing and operating high quality applications. This is because each application is wildly different from all others. However, there are tools which can make DevOps a far less daunting proposition. One of these is known as Azure DevOps. 
+
+# Azure DevOps
+
+Azure DevOps has a long pedigree. It can trace its roots back to when Team Foundation Server first move online and through the various name changes: Visual Studio Online and Visual Studio Team Services. Through the years, however, it has become far more than its predecessors. 
+
+Azure DevOps is divided into five major components: Boards, Repos, Pipelines, Test Plans and Artifacts. 
+
+![The five major areas of Azure DevOps](media/devops-components.png)
+
+**Azure Boards** - Provides an issue and work item tracking tool which strives to allow users to pick the workflows which work best for them. It comes with a number of pre-configured templates including ones to support SCRUM and Kanban styles of development. 
+
+**Azure Repos** - Source code management which supports the venerable Team Foundation Version Control (TFVC) as well as the industry favorite git. Pull requests provide a way to enable social coding by fostring discussion of changes as they are made. 
+
+**Azure Pipelines** - A build and relese management system which supports tight integration with Azure. Builds can be run on a variety of platforms from Windows to Linux to MacOS. Build agents may be provisioned in the cloud or on premise. 
+
+**Azure Test Plans** - No QA person will be left behind with the test management and exploratory testing support offered by the Test Plans Features.
+
+**Azure Artifacts** - An artifact feed which allows companies to create their own, internal, versions of nuget, npm and others. It serves a double purpose of acting as a cache of upstream packages in the event of failure of a centralized repository.
+
+The top-level container in Azure DevOps is known as a Project. The components can be turned on and off for any project. If users want to manage their source code in GitHub but still take advantage of Azure Pipelines then that's perfectly possible. In fact, many open source projects have taken advantages of the [free builds](https://azure.microsoft.com/en-ca/blog/announcing-azure-pipelines-with-unlimited-ci-cd-minutes-for-open-source/) offered by Azure DevOps while keeping their source code on GitHub. Some significant open source project such as [Visual Studio Code](https://code.visualstudio.com/), [yarn](https://yarnpkg.com/en/), [gulp](https://gulpjs.com/) and [NumPy](https://www.numpy.org/) have made the transition. 
```

Comment:
container might be confusing; maybe unit of organization / organizational unit?
The components... what components?
taken advantages -> advantage
project such as -> projects